app/views/users/my_profile.erb

```diff
@@ -1,3 +1,12 @@
 <h1><%= current_user.full_name %> profile </h1>
 
+<table class="table-striped table-hover">
+  <thead>
+    <th> Statistics: </th>
+  </thead>
+  <tbody>
+
```

Comment:
empty table body with new line :) but we can leave it for now, I assume you know what you're doing 💃